src/org/omegat/core/dictionaries/LingvoDSL.java

```diff
@@ -130,4 +134,91 @@ private void loadData(Stream<String> stream) {
                     .collect(Collectors.toList());
         }
     }
+
+    @SuppressWarnings("visibilitymodifier")
+    static class RE {
```

Comment:
I'm not sure why we need this class. Can't we just use a `HashMap<Pattern, String>` to store the patterns as keys and replacements as values?

If order is important it can be a `TreeMap`.